packages/frontend/src/components/message/Message.tsx

```diff
@@ -271,6 +272,15 @@ function buildContextMenu(
   // Do not show "reply" in read-only chats
   const showReply = chat.canSend
 
+  // See https://github.com/deltachat/deltachat-desktop/issues/4695.
+  const showEdit =
+    message.fromId === C.DC_CONTACT_ID_SELF &&
+    message.text !== '' &&
+    chat.canSend &&
+    !message.isInfo &&
+    message.viewType !== 'VideochatInvitation' &&
+    !message.hasHtml
```

Comment:
if we add a condition here like
`&& runtime.getRC_Config().devmode`
we could merge this PR and "activate" it once it should go public